src/pharmpy/tools/structsearch/tmdd.py

```diff
@@ -75,14 +75,50 @@ def create_remaining_models(model, ests, num_peripherals_qss, dv_types):
     if num_peripherals_qss < num_peripherals_model:
         model = remove_peripheral_compartment(model)
 
-    models = (
-        create_full_models(model, ests, dv_types)
-        + create_cr_models(model, ests, dv_types)
-        + create_ib_models(model, ests, dv_types)
-        + create_crib_models(model, ests, dv_types)
-        + create_wagner_model(model, ests, dv_types)
-        + create_mmapp_model(model, ests, dv_types)
-    )
+    if dv_types is None:
+        models = (
+            create_full_models(model, ests, dv_types)
+            + create_cr_models(model, ests, dv_types)
+            + create_ib_models(model, ests, dv_types)
+            + create_crib_models(model, ests, dv_types)
+            + create_wagner_model(model, ests, dv_types)
+            + create_mmapp_model(model, ests, dv_types)
+        )
+    else:
+        if len(dv_types) < 2:
```

Comment:
Does this stop having only drug concentation? Shouldn't that be supported?

Reply:
if `dv_types` is set to `None` then everything is treated as drug. If we only use drug in `dv_types` then everything will be treated as drug as well, not just rows where DVID == 1. But a user might think only rows with DVID == 1 will be used.